improc/alignment.py

```diff
@@ -350,7 +350,14 @@ def _align_swarp( self, image, target, sources, target_sources ):
             #  For most images I've seen, no image
             #  will have a pixel with noise above 100000,
             #  hence the 1e-10.
-            warpedim.flags[ warpedim.weight < 1e-10 ] = 1
+            # WH: to get the out of bounds pixels from the alignment
+            #  flag all such pixels that were not already flagged bad
+
+            flags_before = np.logical_or(warpedim.flags == 0, warpedim.flags == 8).sum()
+            warpedim.flags[ np.logical_and(warpedim.flags == 0, warpedim.weight < 1e-10)] = 8
+            flags_after = np.logical_or(warpedim.flags == 0, warpedim.flags == 8).sum()
+            assert flags_before == flags_after, "Something besides 0 or 8 was changed"
```

Comment:
I would avoid assert statements outside of tests (this is mostly style).

Reply:
It looks good to me, I wouldn't bother with this extra precaution test in the final code version.

Reply:
Please make sure to replace `8` with the converter function for `out of bounds`.